### PR TITLE
readd 94f1ac996c0eb6f150d99f53f22a16c58c8ca3a4

### PR DESCRIPTION
**Pull request by semool**

### CODE REVIEW

File: language/English/strings.po [hunk 1]
```diff
@@ -267,51 +268,81 @@ msgid "Show now playing info"
 msgstr ""
 
 msgctxt "#31156"
-msgid "[B]Configure Appearance Settings[/B][CR][CR] •     Change the skin[CR] •     Set language and region[CR] •     Change file listing options[CR] •     Set up a screensaver"
+msgid ""
```
Comment (beatmasterRS) [inline language/English/strings.po:271]: Whats the reason for this?

Comment (semool) [reply language/English/strings.po]: That was Poedit :) i dont checked the settings after a update. Its normal when you make line breaks in the Translations. I have changed the settings back to not add linebreaks automaticly.

**Comment by beatmasterRS** [comment]
Thanks a lot for your work :)

For string #37628 we need another translation, cause the german translation "verschoben" exists also for view 509. 
Or we use another name for the view type 513. What do you think?

**Comment by semool** [comment]
Hmm we can use the direct Translation "Mehr verschoben" ?

**Comment by beatmasterRS** [comment]
That would be possible, of course. But in the meantime I think that a name that better describes this view type would be better. This view is the only vertical view except for the list views. What do you think about "vertical shifted"? Something with "vertical" would be good is thin. I dont know :D

**Comment by semool** [comment]
Vertical Shifted sounds good :)

**Pull request merged**